src/control/ros/RosTrajectoryExecutor.cpp

```diff
@@ -122,7 +122,7 @@ void RosTrajectoryExecutor::validate(const trajectory::Trajectory* traj)
         "Trajectory StateSpace incompatible with MetaSkeleton");
   }
 
-  // TODO: No check's happening here. Is that correct?
+  // TODO(egordon): check joints in traj against this->getJoints()
```

Comment:
Has this comment been addressed?

Reply:
Removed, this is actually no longer needed. The trajectory has no concept of DoFs directly. It operates on a StateSpace. If the trajectory is compatible with the metaskeleton (checked in the previous line), and the MetaSkeleton DoFs are registered (done in the Constructor), then by definition the trajectory DoFs are compatible with the registered DoFs.